libkpod/container_server.go

```diff
@@ -590,6 +597,46 @@ func (c *ContainerServer) ListContainers() []*oci.Container {
 	return c.state.containers.List()
 }
 
+// ListRunningContainers returns a list of all containers that are currently running
+func (c *ContainerServer) ListRunningContainers() []*oci.Container {
```

Comment:
I started to do that but I ran into an issue: If ListContainers is a variadic function accepting zero or more filter functions, each taking in an *oci.Container and returning a bool, we can make things work for every single filter except for a slice of ids or names.  Is there a way around this aside from changing the function header from `ListContainers(filters ...func(*oci.Container) bool)` to `ListContainers(idsOrNames []string, filters ...func(*oci.Container) bool)`?